src/DataTable/CardView.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useRows } from './hooks';
+import { CardGrid } from '..';
+
+const CardView = ({
+  tableName, columnSizes, CardComponent,
+}) => {
+  if (!tableName) {
+    return null;
+  }
+
+  const {
+    getTableProps, prepareRow, displayRows,
+  } = useRows(tableName);
+
+  const renderCards = () => displayRows.map((row) => {
+    prepareRow(row);
+    return (
+      <CardComponent {...row} key={row.id} />
+    );
+  });
+
+  if (!getTableProps) {
+    return null;
+  }
+
+  return (
+    <CardGrid
+      columnSizes={columnSizes}
+    >
+      {renderCards()}
+    </CardGrid>
+  );
+};
```

Comment:
This is very cool. Riffing on the idea: What if you didn't need this component at all. Could a generic `TableItemsList` with props `as` and `tableItemAs` afford for the regular Table rows by default or:

```
<TableContextProvider>
  <DataTable>
     <TableControlBar>
     <TableItemList as={CardGrid} tableItemAs={MiyazakiCard} />
  </DataTable>
</TableContextProvider>
```

Reply:
oooo, fancy. Let me think about it / play around with that idea.

Reply:
I think this is for a subsequent iteration. 😄